rocAL/rocAL/source/augmentations/color_augmentations/node_color_twist.cpp

```diff
@@ -45,33 +43,47 @@ void ColorTwistBatchNode::create_node()
     _hue.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
     _sat.create_array(_graph , VX_TYPE_FLOAT32, _batch_size);
 
-    _node = vxExtrppNode_ColorTwistbatchPD(_graph->get(), _inputs[0]->handle(), _src_roi_width, _src_roi_height, _outputs[0]->handle(), _alpha.default_array(), _beta.default_array(), _hue.default_array(), _sat.default_array(), _batch_size);/*A temporary fix for time being*/
+    // if(_inputs[0]->info().layout() == RocalTensorlayout::NCHW)
+    //     _layout = 1;
+    // else if(_inputs[0]->info().layout() == RocalTensorlayout::NFHWC)
+    //     _layout = 2;
+    // else if(_inputs[0]->info().layout() == RocalTensorlayout::NFCHW)
+    //     _layout = 3;
+    if(_inputs[0]->info().roi_type() == RocalROIType::XYWH)
```

Comment:
Why this roi assignment is different from crop?

Reply:
Resolved